DESCRIPTION

```diff
@@ -37,7 +37,7 @@ Suggests:
     quanteda (>= 3.0),
     testthat,
     covr
-URL: https://github.com/quanteda/readtext
+URL: https://github.com/quanteda/readtext, https://readtext.quanteda.io/
```

Comment:
@kbenoit All the rest looks good! I'd still recommend adding this line. For discoverability, https://www.r-bloggers.com/2019/12/about-urls-in-description/